pipit/graph.py

```diff
@@ -109,3 +120,19 @@ def add_root(self, node):
 
     def get_node(self, calling_context_id) -> "Node":
         return self.calling_context_id_map.get(str(calling_context_id))
+
+    def is_empty(self):
+        if not self.roots:
+            return True
+        else:
+            return False
+
+    def get_graph_helper(self, node, level=0):
+        ret = "\t" * level + node.name + "\n"
+        for i in node.get_child():
+            ret += self.get_graph_helper(i, level + 1)
+        return ret
+
+    def get_graphs(self):
+        for i in self.roots:
+            return self.get_graph_helper(i)
```

Comment:
If there are multiple roots, won't this only return the graph for the first one?

Reply:
Yes. Going to fix it.